src/Bedrock.Framework.Experimental/Protocols/RabbitMQ/Methods/ConnectionOpen.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Buffers;
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Bedrock.Framework.Experimental.Protocols.RabbitMQ.Methods
+{
+    public class ConnectionOpen : MethodBase, IAmqpMessage
+    {
+        public override byte ClassId => 10;
+        public override byte MethodId => 40;
+
+        public ReadOnlyMemory<byte> Vhost { get; private set; }
+        public ReadOnlyMemory<byte> Reserved1 { get; private set; }
+
+        public byte Reserved2 { get; }
+
+        public ConnectionOpen(ReadOnlyMemory<byte> vhost, ReadOnlyMemory<byte> reserved1, byte reserved2)
+        {
+            Vhost = vhost;
+            Reserved1 = reserved1;
+            Reserved2 = reserved2;
+        }
+
+        public bool TryParse(in ReadOnlySequence<byte> input,  out SequencePosition end)
+        {
+            throw new NotImplementedException();
+        }
+
+        public void Write(IBufferWriter<byte> output)
+        {
+            int PayloadLength = 1+Vhost.Length + 1+Reserved1.Length + MethodHeaderLength + sizeof(byte);
```

Comment:
```suggestion
            var payloadLength = 1 + Vhost.Length + 1 + Reserved1.Length + MethodHeaderLength + sizeof(byte);
```

Reply:
damn ! :-)  Maybe there is a shortcut/option in visual studio to avoid this formatting issue ?

Reply:
In VS: Edit -> Advanced -> Format Document

Reply:
i knew this shortcut pour indentation, bracet etc.. but did'nt know for this kind of formating.